src/interface.c

```diff
@@ -36,12 +36,21 @@ void particle_finalise(SEXP r_ptr) {
 SEXP r_particle_run(SEXP r_ptr, SEXP r_step_end) {
   particle *obj = (particle*) read_r_pointer(r_ptr, true);
   size_t step_end = (size_t) INTEGER(r_step_end)[0];
+  
+  // Note: running a single particle allocs a gsl rng each
```

Comment:
This section is awkward, and makes me think that perhaps we should get rid of the single particle interface? Or, as it's a useful test, discourage it?